src/main/java/com/iota/iri/service/API.java

```diff
@@ -1676,7 +1676,7 @@ private synchronized AbstractResponse storeMessageStatement(String address, Stri
                 storeTransactionsStatement(trytes);
             } catch (Exception e) {
                 //transaction not valid
-                return ErrorResponse.create("Invalid trytes input");
+                return ErrorResponse.create("Error: " + e.getMessage());
```

Comment:
I would do it as the issue defined

```suggestion
                return ErrorResponse.create("Invalid trytes input: " + e.getMessage());
```

Reply:
Because you could get `Invalid trytes input: Invalid trytes input`. 
I thought `Error: Invalid trytes input` looks better or `Error: Whatever`

Reply:
ok, accepted